### PR TITLE
Feature/pdp

### PR DESCRIPTION
**Pull request by lusiakpurnama**

### CODE REVIEW

File: src/scss/sections/_product-waitlist.scss [hunk 1]
```diff
@@ -453,3 +453,32 @@
...
 }
+
+.product-waitlist-form {
+	&-oos {
+		border-radius: 20px;
+
+		@include media-breakpoint-down(md) {
+			.input-group .form-control,
+			.input-group .btn {
+				border-radius: $border-radius;
```
Comment (foria) [inline src/scss/sections/_product-waitlist.scss:464]: for border-radius use the default utility class directly on the DOM

File: src/scss/sections/_product-waitlist.scss [hunk 2]
```diff
@@ -453,3 +453,32 @@
...
+			.btn {
+				width: 100%;
+				margin-top: map-get($spacers, 1);
+			}
+		}
+
+		@include media-breakpoint-up(lg) {
+			.input-group {
+				background: $white;
+				border-radius: $border-radius-lg;
```
Comment (foria) [inline src/scss/sections/_product-waitlist.scss:476]: for border-radius use the default utility class directly on the DOM

File: src/scss/sections/_product-waitlist.scss [hunk 3]
```diff
@@ -453,3 +453,32 @@
...
+			}
+			.form-control {
+				border: 0;
+				border-radius: $border-radius-lg;
```
Comment (foria) [inline src/scss/sections/_product-waitlist.scss:480]: for border-radius use the default utility class directly on the DOM

File: src/scss/sections/_product-waitlist.scss [hunk 4]
```diff
@@ -453,3 +453,32 @@
 		right: 8px;
 	}
 }
+
+.product-waitlist-form {
```
Comment (foria) [inline src/scss/sections/_product-waitlist.scss:457]: we need to optimise the code above, please create the element in the DOCS with this version and another dev can work on it

File: src/scss/components/_tabs.scss [hunk 5]
```diff
@@ -1,33 +1,36 @@
 .nav-tabs {
-  border-bottom: none;
+  &--product,
+  &--real-results {
```
Comment (foria) [inline src/scss/components/_tabs.scss:3]: why do we need this nesting?

**Review by foria — CHANGES_REQUESTED** [review]
TO FIX:
1. Product Carousel: chevrons must be primary color, the top and bottom chevron must always be aligned to the top and bottom of the image beside, chevron must disappear when an the begin and end of carousel
2. integrate text next to review stars as in the design
3. review bottom section won't change, can we implement the one we have currently?
4. please review text styles for the promo discount code in order to match design
5. please review style for upsells as well
6. "What makes us different" icons can be hardcoded with SVG

File: src/scss/components/_tabs.scss (hunk 5 above)
Comment (lusiakpurnama) [reply src/scss/components/_tabs.scss:3]: This is what I mentioned before, the tabs css for homepage is affecting the tab in product page, so I asked pebi to adjust it.

File: src/scss/compounds/_product-banner.scss [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+	&__hair &__text {
+		background: $primary-light;
+	}
+	&__body &__text {
+		background: $bali-bod-blue;
+		color: #fff;
+	}
+	&__body-2 &__text {
+		background: $sh-purple-light;
+	}
```
[the remark this answers is not in the document]
Comment (lusiakpurnama) [reply src/scss/compounds/_product-banner.scss:17]: will update

File: src/scss/sections/_product-waitlist.scss (hunk 1 above)
Comment (lusiakpurnama) [reply src/scss/sections/_product-waitlist.scss:464]: border radius for desktop and mobile is different, so cannot directly using the utility class

File: src/scss/sections/_product-waitlist.scss (hunk 2 above)
Comment (lusiakpurnama) [reply src/scss/sections/_product-waitlist.scss:476]: border radius for desktop and mobile is different, so cannot directly using the utility class

File: src/scss/sections/_product-waitlist.scss (hunk 3 above)
Comment (lusiakpurnama) [reply src/scss/sections/_product-waitlist.scss:480]: border radius for desktop and mobile is different, so cannot directly using the utility class

File: src/scss/core/_vars.scss [hunk 7]
```diff
@@ -88,6 +88,9 @@ $gray-900: $body-color;
 $text-muted: $gray-500;
 $border-color: $gray-400;
 
+// Border
+$border-radius-lg: calc-em(8, $unit: 1rem);
```
Comment (foria) [inline src/scss/core/_vars.scss:92]: border-radius must be the same on all breakpoints

**Review by foria — CHANGES_REQUESTED** [review]

File: src/partials/components/product-waitlist-oos.hbs [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<form id="product-waitlist-form-oos" class="product-waitlist-form-oos bg-primary-light px-g py-3 {{ class }}">
```
[the remark this answers is not in the document]
Comment (lusiakpurnama) [reply src/partials/components/product-waitlist-oos.hbs:1]: already in the compounds

File: src/partials/components/product-image-carousel.hbs [hunk 9]
```diff
@@ -8,15 +8,15 @@
 	</figure>
 </div>
 
-<div id="product-image-carousel__indicator{{#if num}}{{num}}{{/if}}" class="product-image-carousel__indicator col-12 col-lg-1 order-lg-1 carousel slide carousel--loop mb-1" {{#if (gt totalSlide 5)}}data-slide-number="5" data-ride="carousel" data-interval="false"{{/if}}>
+<div id="product-image-carousel__indicator{{#if num}}{{num}}{{/if}}" class="product-image-carousel__indicator col-12 col-lg-1 order-lg-1 carousel slide carousel--loop mb-1 mb-lg-0 px-0 px-lg-g" {{#if (gt totalSlide 5)}}data-slide-number="5" data-ride="carousel" data-interval="false"{{/if}}>
```
Comment (foria) [inline src/partials/components/product-image-carousel.hbs:11]: the bottom chevron is not aligned with the bottom of the big image, also the top chevron is not visible

File: src/scss/components/_list.scss [hunk 10]
```diff
@@ -1,7 +1,25 @@
...
+			font-family: $supernova-icon-font;
+			content: "\a0#{map-get($supernova-icon-map, "check")}";
+			color: $primary;
+			position: absolute;
+			top: 0;
+			left: - map-get($spacers, 4);
+		}
+	}
+
+	li:not(:last-child) {
```
Comment (foria) [inline src/scss/components/_list.scss:18]: this can stay inside LI

File: src/scss/components/_product-variant.scss [hunk 11]
```diff
@@ -1,14 +1,34 @@
...
-			left: calc-em(20);
-			transform: translateY(-50%);
+			top: calc-em(40, $unit: 1rem);
+			left: calc-em(20, $unit: 1rem);
+			border-color: $gray-500;
 		}
 	}
 
-	.custom-control-input:checked~.custom-control-label {
+	.custom-control-input:not(:disabled):active ~ .custom-control-label::before {
```
Comment (foria) [inline src/scss/components/_product-variant.scss:11]: please check Sass syntax, nest code related to .custom-control-input

**Review by foria — CHANGES_REQUESTED** [review]

**Pull request merged**